tests/specimen/tumourGrade.test.js

```diff
@@ -10,247 +10,146 @@ const name = 'tumour_grade';
     true,
     loadObjects(specimen, {
-      tumour_grading_system: ' gleason ',
-      tumour_grade: 'Gleason 2–6: The tumor tissue is well differentiated',
+      tumour_grading_system: 'who grading system for cns tumours ',
+      tumour_grade: 'iii',
     }),
   ],
   [
-    'Grading system set to "Gleason", tumour grade is "Gleason 7: The tumor tissue is moderately differentiated"',
+    'Grading system set to "WHO grading system for CNS tumours", tumour grade is "IV"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'gleason',
-      tumour_grade: 'Gleason 7: The tumor tissue is moderately differentiated',
+      tumour_grading_system: 'WHO grading system for cns tumours',
+      tumour_grade: 'IV',
     }),
   ],
   [
-    'Grading system set to "Gleason", tumour grade is "Gleason 8–10: The tumor tissue is poorly differentiated or undifferentiated"',
+    'Grading system set to "FNCLCC grading system", tumour grade is "G1"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'gleason',
-      tumour_grade: 'Gleason 8–10: The tumor tissue is poorly differentiated or undifferentiated',
+      tumour_grading_system: 'FNCLCC grading system',
+      tumour_grade: 'G1',
     }),
   ],
   [
-    'Grading system set to "Gleason", tumour grade is a value from another codelist',
+    'Grading system set to "Gleason grade group system", tumour grade is a value from another codelist',
     false,
     loadObjects(specimen, {
-      tumour_grading_system: 'gleason',
-      tumour_grade: 'Grade IV',
+      tumour_grading_system: 'Gleason grade group system',
+      tumour_grade: 'IV',
     }),
   ],
   [
-    'Grading system set to "Nottingham", tumour grade is "G1 (Low grade or well differentiated)"',
+    'Grading system set to "Gleason grade group system ", tumour grade is "5"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'nottingham ',
-      tumour_grade: 'G1 (Low grade or well differentiated)',
+      tumour_grading_system: 'Gleason grade group system ',
+      tumour_grade: '5',
     }),
   ],
   [
-    'Grading system set to "Nottingham", tumour grade is "G2 (Intermediate grade or moderately differentiated)"',
+    'Grading system set to "Scarff-Bloom-Richardson grading system", tumour grade is "G2"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'nottingham ',
-      tumour_grade: 'G2 (Intermediate grade or moderately differentiated)',
+      tumour_grading_system: 'Scarff-Bloom-Richardson grading system ',
+      tumour_grade: 'G2',
     }),
   ],
   [
-    'Grading system set to "Nottingham", tumour grade is "G3 (High grade or poorly differentiated)"',
+    'Grading system set to "Nuclear grading for DCIS", tumour grade is "G3"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'nottingham ',
-      tumour_grade: 'G3 (High grade or poorly differentiated)',
+      tumour_grading_system: 'Nuclear grading for DCIS ',
```

Comment:
the tests here are missing the word system

Reply:
Thanks Wajiha. I fixed it (added `grading` back in)